scripts/initialize.sh

```diff
@@ -1,32 +1,76 @@
 #!/bin/bash
-set -e pipefail
+set -euo pipefail
+
+LOG_START='\n\e[1;36m'  # new line + bold + cyan
+LOG_END='\n\e[0m'       # new line + reset
+DONE_START='\n\e[1;32m' # new line + bold + green
+DONE_END='\n\n\e[0m'    # new line + reset
 
-# Dafault config files directory.
 CONFIG_DIR_PATH_DEFAULT=$(realpath -m $(dirname $0)/../configs)
+KEEP_ECDSA_PATH=$(realpath $(dirname $0)/../)
+KEEP_ECDSA_SOL_PATH=$(realpath $KEEP_ECDSA_PATH/solidity)
 
-# Read user config file path.
-read -p "Enter path to keep-ecdsa config files directory [$CONFIG_DIR_PATH_DEFAULT]: " config_dir_path
-CONFIG_DIR_PATH=${config_dir_path:-$CONFIG_DIR_PATH_DEFAULT}
+# Defaults, can be overwritten by env variables/input parameters
+NETWORK_DEFAULT="local"
```

Comment:
Shouldn't `CONFIG_DIR_PATH_DEFAULT` be placed here, under `# Defaults, can be overwritten...`?

Reply:
045db58